### PR TITLE
Refactor 4

### PR DESCRIPTION
**Pull request by sgoraczynski**
https://github.com/sgoraczynski/canvas-helmet/issues/4

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -1 +1,2 @@
 [1124/122051.086:ERROR:crash_report_database_win.cc(457)] unexpected header
+[1208/112014.877:ERROR:crash_report_database_win.cc(455)] unexpected header
```
Comment (pbrudny) [inline debug.log]: It is better not to have the log file in the repository. Please use .gitignore for it.

File: canvashelmet.html [hunk 2]
```diff
@@ -7,6 +7,8 @@
         var canvas = document.getElementById('canvas');
         if (canvas.getContext) {
             var ctx = canvas.getContext('2d');
+            var xCoordinate = 132;
```
Comment (pbrudny) [inline canvashelmet.html:10]: Nice one 👍

**Review by pbrudny — CHANGES_REQUESTED** [review]
Looking good to me. However you should remove the log file.

File: debug.log (hunk 1 above)
Comment (sgoraczynski) [reply debug.log]: What is a .gitignore ?

Comment (sgoraczynski) [reply debug.log]: Done

File: canvashelmet.html (hunk 2 above)
Comment (sgoraczynski) [reply canvashelmet.html:10]: Thanks

**Review by pbrudny — APPROVED** [review]

**Pull request merged**